app/react/src/client/testing/index.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  composeStory as originalComposeStory,
+  composeStories as originalComposeStories,
+} from '@storybook/store';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { composeConfigs } from '@storybook/preview-web';
```

Comment:
@tmeasday `composeConfigs` seems like a good candidate to be moved to a csf specific package (for now @storybook/store/csf) rather than be inside of preview-web. WDYT?

Reply:
Sounds good @yannbf. One little wrinkle is the handling of the `renderToDOM` annotation, which I guess technically is "web only" -- I'm not quite sure how best to handle it:

 - In a non-web context (e.g. React Native), that annotation wouldn't be allowed
 - In a web context, we would want to guarantee the annotation existed at the project-level, and make it required on the type.

This is similar to the `canvasElement: HTMLElement` that we added to the `StoryContext` for the play function. In that case [we ended up making it required](https://github.com/ComponentDriven/csf/blob/835a408711df516e2dfaa17d0724da42ec691df8/src/story.ts#L102), so I guess let's follow that pattern for now? I'm honestly unsure.

Reply:
Thanks @tmeasday! I ended up not using that anymore, I didn't make sense for this use case (the composing is really just merging an object with default render function.